### PR TITLE
Gui update

### PR DESCRIPTION
**Pull request by YoungBrendanM**
# Pull Request Template

## Description
Added GUI elements
[comment]: <> (Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change?)

## Type of change

[comment]: <> (Please delete options that are not relevant.)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
See Readme
[comment]: <> (Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration)

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by crsellinger — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+
+### If changes were made to the src\main\antlr4\App.g4:
+ 1. Run "mvn generate-sources" from the repository's root directory
+ 2. Copy all files from target\generated-sources\antlr4\ to src\main\java\parser
+ 3. Add "package parser;" to the first line of the four java class files (This is due to being unable to figure how to import them from the generated-sources folder).
+ 4. Run "mvn package" from the repository's root directory
+ 5. parser-1.0.0-shaded.jar should be created in the target folder.
+
+### If no changes were made to the src\main\antlr4\App.g4:
+ 1. Run mvn package from the repository's root directory
```
Comment (crsellinger) [inline README.md]: "mvn package"

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+## How to Run: 
+### Requirements:
+ 1. Your computer needs to be able to run java code newer than Java 7.
+ 2. Be able to use the java command from command prompt.
+ 3. A copy of parser-1.0.0-shaded.jar after project has been built.
+
+File is selected after program start
+### To run without terminal:
+  1. Double click on the file parser-1.0.0-shaded.jar
+
```
Comment (crsellinger) [inline README.md:42]: 2. Locate the file you'd like to parse using the GUI window and double click.

**Comment by YoungBrendanM** [comment]
[python_test_code (2).txt](https://github.com/YoungBrendanM/CS-4450ParserProject/files/7731223/python_test_code.2.txt)

**Review by crsellinger — COMMENTED** [review]
Suggested changes to the READ.me instructions, not much

**Pull request merged**